src/styles/components/Sponsor.module.css

```diff
@@ -1,49 +1,57 @@
 .container {
   display: flex;
-  padding: 0.5rem;
-  width: 9rem;
+  justify-content: center;
+  align-items: center;
+  width: 14rem;
   height: 8rem;
   border-radius: 1rem;
   background-color: #ffffff;
   position: relative;
   margin: 1rem;
+  z-index: 3;
   cursor: pointer;
-  display: flex;
-  border: none;
+  border: 4px solid var(--wcs-pink);
+}
+
+.container::before {
+  content: '';
+  position: absolute;
+  transform: translateY(-50%);
+  top: 50%;
+  left: 0.25rem; 
+  z-index: 4;
+  width: 0.25rem; 
+  height: 2rem; 
+  background-color: var(--wcs-pink);  
 }
 
 .container::after {
   content: '';
   position: absolute;
   right: -1rem;
   top: -1rem;
-  border-radius: 100%;
-  width: 3rem;
-  height: 3rem;
-  display: flex;
-  border: none;
+  z-index: 2;
+  border-radius: 50%;
+  width: 1.5rem; 
+  height: 1.5rem;
 }
 
 .image {
-  justify-self: center;
-  align-self: center;
-  max-width: 8rem;
-  max-height: 7rem;
+  max-width: 13rem; 
+  max-height: 8rem;
+  position: absolute;
   object-fit: contain;
-}
-
-.platinum::after {
-  background-color: #337ab7;
+  z-index: 1;
+  padding-left: 30px;
+  padding-right: 20px;
+  padding-top: 20px;
+  padding-bottom: 20px;
 }
 
 .gold::after {
```

Comment:
keep the colors for silver and platinum for future use (even if they are not used), and try to have them all be the same type (all hex codes, probably)